.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,34 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          container-runtime: remote
```

Comment:
Isn't this set by default? As it's the only accepted value now?

Reply:
more of a failsafe, but I can drop it if you want - it's the default currently